lib/venice/receipt.rb

```diff
@@ -71,6 +78,13 @@ def initialize(json = {})
         []
       end
 
+      @pending_renewal_info = []
+      if original_json_response && original_json_response['pending_renewal_info']
+        original_json_response['pending_renewal_info'].each do |pending_renewal_attributes|
```

Comment:
`@pending_renewal_info = original_json_response['pending_renewal_info'].map { |attrs| PendingRenewalInfo.new(attrs) } ` would be a shorter verison of the same I think

Reply:
done